src/date-range-picker/index.tsx

```diff
@@ -121,6 +105,12 @@ const DateRangePicker = React.forwardRef(
     }: DateRangePickerProps,
     ref: Ref<DateRangePickerProps.Ref>
   ) => {
+    const { formatValue } = useDateRangePicker({
```

Comment:
Can this function not be exported separately. It seems weird to have a hook here that creates unnecessary and unused state.